src/translations/en.json

```
@@ -4096,7 +4096,9 @@
         "setup": {
           "next": "Next",
           "back": "Back",
-          "done": "Show me my energy dashboard!"
+          "done": "Show me my energy dashboard!",
+          "step": "Step",
+          "of": "of"
```

```suggestion
          "step_of": "of"
```
I would slightly prefer that so it more closely ties to the "step" key.

---

Done